nautobot/ipam/tests/test_filters.py

```diff
@@ -674,6 +672,71 @@ def test_vm_interfaces(self):
             self.queryset.filter(vm_interfaces__in=[vm_interfaces[0], vm_interfaces[1]]),
         )
 
+    def test_has_interface_assignments(self):
+        params = {"has_interface_assignments": True}
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset(params, self.queryset).qs,
+            self.queryset.filter(Q(interfaces__isnull=False) | Q(vm_interfaces__isnull=False)),
+        )
+        params = {"has_interface_assignments": False}
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset(params, self.queryset).qs,
+            self.queryset.filter(Q(interfaces__isnull=True) & Q(vm_interfaces__isnull=True)),
+        )
+
+    def test_present_in_vrf(self):
```

Comment:
Huh. Did we just not have any tests for `present_in_vrf` before?

Reply:
Yeah I dont think so.